### ISSUE TITLE
Move from size aesthetic to linewidth as size is deprecated.

### ISSUE THREAD

**Issue by sims1253** [stan-dev/bayesplot#335]
Since ggplot2 3.4.0 the size aesthetic for lines is deprecated and linewidth should be used instead. As size is absolute and linewidth is relative, the 1 to 1 plot comparison tests will most likely fail as it'll be hard to find the exact linewidth equivalent of the current size values.

**Comment by jgabry** [comment]
Yeah there are probably still some cases of this that need to be changed. I think a bunch of cases have already been updated to `linewidth` (at least on GitHub, maybe not in the CRAN version yet), but not all. If we can get the sizes close enough that the plots don't change substantially that's fine. It will break the visual tests, but that's probably unavoidable, like you say, and we can reset them.
